src/components/search-bar.js

```diff
@@ -18,10 +16,10 @@ export default function SearchBar({ initialValue, autoFocus, onSearch }) {
   };
 
   const handleSearch = value => {
-    if (value.length < MIN_CHAR_COUNT) {
+    if (!value.length) {
       Modal.error({
         title: t('common:error'),
-        content: t('common:searchTextTooShort', { minCharCount: MIN_CHAR_COUNT }),
+        content: t('common:searchTextTooShort', { minCharCount: 1 }),
```

Comment:
Maybe we should change the validation to say that a value is required. Seems a bit weird and inconsistent this way now.

Reply:
Agreed, I created a more specific validation message.